.github/workflows/uffizzi-build.yml

```diff
@@ -3,9 +3,14 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, closed]
     paths-ignore:
+      - '.changeset/**'
+      - 'contrib/**'
+      - 'docs/**'
       - 'microsite/**'
+      - 'scripts/**'
       - 'storybook/**'
-      - 'contrib/**'
+      - '**/*.test.*'
+      - '**/package.json'
```

Comment:
really?

Reply:
i guess if it's worth testing, it'll have also affected yarn.lock which isn't ignored

Reply:
Yep intention here was just avoid metadata changes and let `yarn.lock` pick up the important stuff